code/processes/segmentedtp.q

```diff
@@ -0,0 +1,37 @@
+// Segmented TP process
+// Contains all TP functionality with additional flexibility
+// Configurable logging and subscriptions
+
+.proc.loadf[getenv[`KDBCODE],"/common/os.q"];
+.proc.loadf[getenv[`KDBCODE],"/segmentedtp"];
```

Comment:
Don't need this second line 
set .proc.loadprocesscode to true and it will automatically try and load KDBCODE/segmentedtp
(or whatever this process type ends up being set to, e.g. KDBCODE/segmentedtickerplant)